src/index.ts

```diff
@@ -9,112 +9,119 @@ interface ICoordinates {
   z: number;
 }
 
-interface IOptions {
-  element: HTMLElement;
-  minZoom?: number;
-  maxZoom?: number;
-  captureWheel?: boolean;
-}
-
-export function create({ element, minZoom = MIN_ZOOM, maxZoom = MAX_ZOOM, captureWheel }: Readonly<IOptions>) {
-  const touch = isTouch();
-
-  let action: Action = 'none';
-  let state: Readonly<ICoordinates> = DEFAULT_STATE;
-  const current: ICoordinates & { range: number } = { x: 0, y: 0, z: 0, range: 0 };
-
-  // Update local state & call external set function
-  function setState(value: Readonly<ICoordinates>) {
-    state = value;
-    const point = element.children.item(0);
+export class PinchZoomPan {
```

Comment:
why you have export class PinchZoomPan twice? 
in this file and lib/index.js

Reply:
Because lib - it's a build directory
So you have the same files
We need it because we haven't npm package with our custom code for this repo, and using github URL to fetch this lib